tests/query/test_query_builder.py

```diff
@@ -683,6 +683,15 @@ def test_where_shortcut(self):
         )
         self.assertEqual([1, 'foo'], builder.get_bindings())
 
+    def test_multiple_wheres_in_list(self):
+        builder = self.get_builder()
+        builder.select('*').from_('users').where([['name', '=', 'bar'], ['age', '=', 25]])
+        self.assertEqual(
+            'SELECT * FROM "users" WHERE ("name" = ? AND "age" = ?)',
+            builder.to_sql()
+        )
+        self.assertEqual(['bar', 25], builder.get_bindings())
+
```

Comment:
We should also test the case where conditions are invalid, to see if the exception is actually raised.